internal/backend/auth/authloginhttpsvc/web/descope.html

```diff
@@ -31,6 +31,7 @@ <h1>Descope Login</h1>
         const wcElement = document.getElementsByTagName("descope-wc")[0];
 
         const onSuccess = (e) => {
+            document.cookie = `DSR=${e.detail.refreshJwt}; path=/`;
```

Comment:
maybe just perform a post request to the backend to exchange the token (instead of storing it in a non secure cookie)

post would set the cookie to our jwt, using secure same site attributes
and then you can redirect again